kstreams-live-update/temperature-aggregator/src/main/java/io/debezium/examples/kstreams/liveupdate/aggregator/Main.java

```diff
@@ -1,102 +1,55 @@
-        final String childrenTopic = args[1];
-        final String bootstrapServers = args[2];
-
-        Properties props = new Properties();
-        props.put(StreamsConfig.APPLICATION_ID_CONFIG, "streaming-aggregates-ddd");
-        props.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServers);
-        props.put(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, 10*1024);
-        props.put(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, 1000);
-        props.put(CommonClientConfigs.METADATA_MAX_AGE_CONFIG, 500);
-        props.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
-
-        StreamsBuilder builder = new StreamsBuilder();
-
-        Serde<Long> longKeySerde = new LongKeySerde();
-
-        Serde<TemperatureMeasurement> temperatureMeasurementsSerde = new TemperatureMeasurementSerde();
-        Serde<Station> stationSerde = new StationSerde();
-        Serde<CountAndSum> countAndSumSerde = new CountAndSumSerde();
-
-        KTable<Long, Station> stations = builder.table("dbserver1.inventory.stations", Consumed.with(longKeySerde, stationSerde));
-
-        KTable<String, String> maxTemperaturesByStation = builder.stream(
-                "dbserver1.inventory.temperature_measurements",
-                Consumed.with(longKeySerde, temperatureMeasurementsSerde)
-            )
-            .selectKey((k, v) -> v.stationId)
-            .join(
-                    stations,
-                    (value1, value2) -> {
-                        value1.stationName = value2.name;
-                        return value1;
-                    },
-                    Joined.with(Serdes.Long(), temperatureMeasurementsSerde, null)
-             )
-            .selectKey((k, v) -> v.stationName)
-            .groupByKey(Serialized.with(Serdes.String(), temperatureMeasurementsSerde))
-            .aggregate(
-                    () -> new CountAndSum(), /* initializer */
-                    (aggKey, newValue, aggValue) -> {
-                        aggValue.count++;
-                        aggValue.sum += newValue.value;
-                        return aggValue;
-                    },
-                    Materialized.with(Serdes.String(), countAndSumSerde)
-            )
-            .mapValues(v -> {
-                BigDecimal avg = BigDecimal.valueOf(v.sum / (double) v.count);
-                avg = avg.setScale(1, RoundingMode.HALF_UP);
-                return avg.doubleValue();
-            })
-            .mapValues(String::valueOf);
-
-
-        maxTemperaturesByStation.toStream().to("average_temperatures_by_station", Produced.with(Serdes.String(), Serdes.String()));
-        maxTemperaturesByStation.toStream().print(Printed.toSysOut());
-
-        final KafkaStreams streams = new KafkaStreams(builder.build(), props);
-        Runtime.getRuntime().addShutdownHook(new Thread(streams::close));
-
-        streams.start();
+  public static void main(String[] args) {
+
+    if (args.length != 1) {
```

Comment:
Thanks for cleaning this up btw!

First missed it because all the indentation changed ;)